<launch>
  <node name="probe" pkg="probe" type="probe" output="screen"/>
	<node pkg="tf" type="static_transform_publisher" name="gantry" args="0.12 -0.025 0 0 0 0 base_link gantry 100"  />
	<node pkg="tf" type="static_transform_publisher" name="gantry_carriage" args="0.3 -0.13 0.365 0 0 0 gantry gantry_carriage 100"  />
	<node pkg="tf" type="static_transform_publisher" name="probe_rail" args="0 -0.1 -0.2 0 0 -.50 gantry_carriage probe_rail 100"  />
</launch>